src/pages/index.vue

```diff
@@ -903,14 +903,15 @@ export default {
       searchQuery: '',
     };
   },
-  async fetch({ store }) {
+  async fetch({ store, route }) {
+    const { t } = route.query;
```

Comment:
What would be the initial value of t and how would it be defined?

Reply:
assume `t` is unix timestamp